frontend/src/app/Settings/Users/UsersSideNav.tsx

```diff
@@ -21,77 +21,86 @@ const UsersSideNav: React.FC<Props> = ({
   users,
   onChangeActiveUser,
 }) => {
+  const getIdsAsString = (users: LimitedUser[]) => {
+    return users.map((user) => "user-tab-" + user.id.toString()).join(" ");
+  };
+
   return (
     <div className="display-block users-sidenav">
-      <h3 className="users-header">Users</h3>
-      <ul className="usa-sidenav">
-        {users.map((user: LimitedUser) => {
-          let statusText;
-          switch (user.status) {
-            case "ACTIVE":
-              statusText = (
-                <span className="sidenav-user-status padding-left-0"></span>
-              );
-              break;
-            case "PROVISIONED":
-              statusText = (
-                <>
-                  <PendingIcon />
-                  <span className="sidenav-user-status">
-                    {formatUserStatus(user.status)}
-                  </span>
-                </>
-              );
-              break;
-            case "SUSPENDED":
-              statusText = (
-                <>
-                  <DeactivatedIcon />
-                  <span className="sidenav-user-status">
-                    {formatUserStatus(user.status)}
-                  </span>
-                </>
-              );
-              break;
-            default:
-              statusText = "";
-          }
-          return (
-            <li
-              className="usa-sidenav__item users-sidenav-item"
-              onClick={() => onChangeActiveUser(user.id)}
-              key={user.id}
-            >
-              <button
-                role="tab"
-                style={{ cursor: "pointer" }}
-                className={classnames(
-                  "usa-button--unstyled",
-                  "text-ink",
-                  "text-no-underline",
-                  "padding-105 padding-right-2 padding-left-3",
-                  activeUserId === user.id && "usa-current"
-                )}
-                aria-selected={activeUserId === user.id}
-                aria-label={displayFullName(
-                  user.firstName,
-                  user.middleName,
-                  user.lastName
-                )}
-              >
-                <div className="sidenav-user-name">
-                  {displayFullName(
+      <h2 className="users-header">Users</h2>
+      <nav className="prime-secondary-nav" aria-label="Tertiary navigation">
```

Comment:
(non-blocking) Maybe this should be a more descriptive label? 🤔

Reply:
yeah I was thinkng about that too! This was a pre-existing label (just looks new b/c I also changed the  header size) and I ultimately decided not to refactor all the header labels 😬

Reply:
I generally agree that we shouldn't refactor all the header labels here, but I think this nav label in particular is bad enough that it's worth changing now (or adding a new ticket for, whichever you prefer @georgehager).

Reply:
@emmastephenson fair enough! I don't think this is worth it's own ticket, so refactored to `users-sidenav-header` to follow the semantic pattern for the sidenav component. ready for re-review @emyl3!

Reply:
Sorry I should have been more clear! I was talking about the `aria-label` on line 31 but yes you are right that was a pre-existing name for an aria-label before your changes. I don't know if that's what you were talking about too @emmastephenson 😆 Anyhoo, this LGTM!

Reply:
my bad, I should have clarified before making the change! I think we could add a refactoring ticket for addressing the `aria-label`-ing for navigation throughout the app, I've seen us use `Secondary` especially as well as `Tertiary` for navigation throughout, and maybe we want to move away from that paradigm to something more descriptive. Thoughts @emmastephenson?

Reply:
I think it feels weird to me to change the paradigm in one place but not everywhere else, that's why I'd advocate for a slightly larger refactor if we want to go the descriptive route.